Apps/Playground/iOS/LibNativeBridge.mm

```diff
@@ -7,10 +7,22 @@
 #import <Babylon/ScriptLoader.h>
 #import <Babylon/XMLHttpRequest.h>
 #import <Shared/InputManager.h>
+#import "Babylon/XMLHttpRequestApple.h"
 
 std::unique_ptr<Babylon::AppRuntime> runtime{};
 std::unique_ptr<InputManager::InputBuffer> inputBuffer{};
 
+void InitializeXMLHttpRequest(Napi::Env env)
+{
+    JSGlobalContextRef globalContext = Napi::GetJavaScriptCoreGlobalContext(env);
+    [[XMLHttpRequest new] extend:globalContext :^(CompletionHandlerFunction completetionHandlerFunction) {
+        runtime->Dispatch([completetionHandlerFunction](Napi::Env env) {
+            completetionHandlerFunction();
+        });
+     }
+     ];
+}
```

Comment:
Why are these functions not part of the plugin? Shouldn't the initialization of plugin be in the plugin?

Reply:
I need runtime. I could pass it as a lambda but it's already intracted enough

Reply:
Passing in runtime is expected for many plugins. Why is this one different?

Reply:
I've spent 2-3hours trying to do the init on the htttprequest side without success. I get a crash I can't explain in the lambda cascaded calls. This makes no sense as the code should just be moved from an .mm to another.

Reply:
Can you put this in a branch? I can try it out also.

Reply:
I pushed into the same branch. I did the change for iOS.